ruby/ext/google/protobuf_c/upb.c

```diff
@@ -14311,7 +14311,7 @@ static bool end_timestamp_zone(upb_json_parser *p, const char *ptr) {
   }
 
   /* Normalize tm */
-  seconds = timegm(&p->tm);
+  seconds = mktime(&p->tm) - timezone;
```

Comment:
This does not work on *BSD platforms where `timezone` is not a long int variable but a [function](https://www.freebsd.org/cgi/man.cgi?query=timezone).

Is this change due to timegm() not available everywhere?